pfcpiface/interface.go

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright(c) 2020 Intel Corporation
+
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"net"
+)
+
+type fastPath interface {
+	exit()
+	setUpfInfo(u *upf, conf *Conf)
+	setInfo(udpConn *net.UDPConn, updAddr net.Addr, pconn *PFCPConn)
```

Comment:
what is this?

Reply:
this function sets the udp conn and pconn for the interface to use. The interface needs the conn information to remove sessions in sendDeleteAllSessionsMsgtoUPF and the udp Conn information is used to send the session report.

Reply:
Typo updAddr. The layering somehow does not feel right.

Reply:
updated